srcs/raycasting/draw_navigator.c

```diff
@@ -0,0 +1,123 @@
+#include "cub.h"
+
+static void	draw_square(t_cub *cub, int x, int y, int color)
+{
+	int i;
+	int	j;
+
+	i = 0;
+	while (i < cub->pixel_per_square)
+	{
+		j = 0;
+		while (j < cub->pixel_per_square)
+		{
+			if (i == 0 || i == cub->pixel_per_square - 1 ||
+				j == 0 || j == cub->pixel_per_square - 1)
+				my_mlx_pixel_put(cub, x + i, y + j, 0x00000000);
+			else
+				my_mlx_pixel_put(cub, x + i, y + j, color);
```

Comment:
I'm wondering to what extent it would be interesting to create a new image pointer and then draw this on top of the current screen with mlx_put_image_to_window?

Perhaps it doesn't matter at all.

Reply:
So... Like what you're doing with the textures? No idea honestly, in one case you're repainting pixels from the same image, on the other one you're putting more images... I don't really know what consumes more computational power, I guess it's the part of putting the image to the screen. I see the logic of what you're saying, it encapsulates things more, which is good. It just forces me to rethink things a bit 😅

Reply:
With the textures implementation I think you're actually right, I might have to do it the way you say